python/lib/scheduler/dmod/scheduler/scheduler.py

```diff
@@ -20,10 +20,10 @@
     from .job import Job
 
 
-class DockerServiceParameters():
-    def __init__(self, image_tag: str = None, constraints: list = [], user: Optional[str] = None, hostname: str = None,
-                 labels: dict = {}, serv_name: str = None, mounts: list = [], env_vars: Optional[Dict[str, str]] = None,
-                 secrets: List[SecretReference] = [], cap_add: Optional[List[str]] = None):
+class DockerServiceParameters:
```

Comment:
Honestly, why dont we just make this a `dataclass` since that is all its used for?

Reply:
Not a bad idea, but I didn't want to get that deep down in the weeds in something I'm not super knowledgeable about. That's a tad out of scope.